core/lib/spree/testing_support/factories/refund_factory.rb

```diff
@@ -2,7 +2,7 @@
   sequence(:refund_transaction_id) { |n| "fake-refund-transaction-#{n}"}
 
   factory :refund, class: Spree::Refund do
-    amount 100.00
+    amount { payment.amount / 2 }
```

Comment:
No reason in particular, I chose arbitrarily since a refund could be for a partial amount.